core/src/main/java/org/infinispan/distribution/ch/ConsistentHash.java

```diff
@@ -41,6 +41,11 @@
  */
 public interface ConsistentHash {
 
+   public static final String STATE_CONSISTENT_HASH = "consistentHash";
+   public static final String STATE_HASH_FUNCTION = "hashFunction";
+   public static final String STATE_MEMBER = "member.%d";
+   public static final String STATE_MEMBERS = "members";
```

Comment:
Actually, since those are common to different CH's it's best if they are moved to a dedicated "constants" class.